infrastructure/modules/armonik/ingress-cert.tf

```diff
@@ -112,16 +141,37 @@ resource "tls_locally_signed_cert" "ingress_client_certificate" {
   ]
 }
 
+resource "pkcs12_from_pem" "ingress_client_pkcs12" {
+  count           = length(tls_locally_signed_cert.ingress_client_certificate)
```

Comment:
You should use a `for_each` with the `tls_locally_signed_cert.ingress_client_certificate` map